### PR TITLE
PathofBuildingCommunity.PathofBuilding version 2.19.2.0

### PR DESCRIPTION
**Pull request by orange-lion**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/68011)

### CODE REVIEW

File: manifests/p/PathofBuildingCommunity/PathofBuilding/2.19.2.0/PathofBuildingCommunity.PathofBuilding.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: PathofBuildingCommunity.PathofBuilding
+PackageVersion: 2.19.2.0
```
Comment (ItzLevvie) [inline manifests/p/PathofBuildingCommunity/PathofBuilding/2.19.2.0/PathofBuildingCommunity.PathofBuilding.installer.yaml:5]:
```suggestion
PackageVersion: 2.19.2
```
This must match Control Panel.

You'll also need to change `manifests/p/PathofBuildingCommunity/PathofBuilding/2.19.2.0` to `manifests/p/PathofBuildingCommunity/PathofBuilding/2.19.2`

File: manifests/p/PathofBuildingCommunity/PathofBuilding/2.19.2.0/PathofBuildingCommunity.PathofBuilding.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: PathofBuildingCommunity.PathofBuilding
+PackageVersion: 2.19.2.0
```
Comment (ItzLevvie) [inline manifests/p/PathofBuildingCommunity/PathofBuilding/2.19.2.0/PathofBuildingCommunity.PathofBuilding.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.19.2
```
This must match Control Panel.

You'll also need to change `manifests/p/PathofBuildingCommunity/PathofBuilding/2.19.2.0` to `manifests/p/PathofBuildingCommunity/PathofBuilding/2.19.2`

File: manifests/p/PathofBuildingCommunity/PathofBuilding/2.19.2.0/PathofBuildingCommunity.PathofBuilding.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: PathofBuildingCommunity.PathofBuilding
+PackageVersion: 2.19.2.0
```
Comment (ItzLevvie) [inline manifests/p/PathofBuildingCommunity/PathofBuilding/2.19.2.0/PathofBuildingCommunity.PathofBuilding.yaml:5]:
```suggestion
PackageVersion: 2.19.2
```
This must match Control Panel.

You'll also need to change `manifests/p/PathofBuildingCommunity/PathofBuilding/2.19.2.0` to `manifests/p/PathofBuildingCommunity/PathofBuilding/2.19.2`

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by orange-lion — COMMENTED** [review]
Hi @ItzLevvie

Thanks for the feedback. Based on what you've said I think the PackageIdentifier should be 'PathofBuildingCommunity.PathofBuildingCommunity' to also reflect Control Panel. Can this just be updated in the manifest or should I close PR and submit a new one?

Thank you.

**Pull request closed without merging**